modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/text/TextDDMFormFieldTypeSettings.java

```diff
@@ -96,6 +101,8 @@
 								"indexType", "labelAtStructureLevel",
 								"localizable", "nativeField", "readOnly",
 								"dataType", "type", "showLabel", "repeatable",
+								"requireConfirmation", "direction",
+								"confirmationLabel", "confirmationErrorMessage",
```

Comment:
@carolmariaabb, can we sort it alphabetically or will it affect the UI behavior?

Reply:
It will affect the UI.